plugins/webp-uploads/tests/test-picture-element.php

```diff
@@ -260,9 +260,16 @@ function ( string $content ): string {
 		);
 	}
 
-	public function test_disable_responsive_image_with_picture_element(): void {
+	/**
+	 * @dataProvider data_provider_test_disable_responsive_image_with_picture_element
+	 *
+	 * @param Closure $add_filter The filter.
+	 */
+	public function test_disable_responsive_image_with_picture_element( ?Closure $add_filter ): void {
 		// Disable responsive images.
-		add_filter( 'wp_calculate_image_sizes', '__return_false' );
+		if ( $add_filter instanceof Closure ) {
+			$add_filter();
+		}
```

Comment:
Per above, this is never null:

```suggestion
		$add_filter();
```